llvm/lib/Bitcode/Reader/BitcodeReader.cpp

```diff
@@ -2128,6 +2128,8 @@ static Attribute::AttrKind getAttrFromCode(uint64_t Code) {
     return Attribute::SanitizeThread;
   case bitc::ATTR_KIND_SANITIZE_MEMORY:
     return Attribute::SanitizeMemory;
+  case bitc::ATTR_KIND_SANITIZE_NUMERICAL_STABILITY:
```

Comment:
Would be possible to extract attr, bitcode into a separate patch?

Reply:
Yeah, i can do that.